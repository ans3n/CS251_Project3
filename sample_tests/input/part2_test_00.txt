int float
insert -909 378.732
insert -204 -9.91974
peek -909
insert 695 258.504
maximum_key
print
size
print
insert 297 268.386
empty
insert -481 -729.899
peek 297
insert 817 405.289
maximum_key
empty
insert 759 220.44
peek -481
size
peek -909
size
minimum_key
insert 849 -213.484
print
insert -833 519.52
print
insert 876 875.023
maximum_key
print
peek 297
insert -651 -184.926
insert -13 266.374
insert -746 -187.403
maximum_key
insert 513 -281.676
peek -833
size
maximum_key
print
insert -279 -10.7584
print
insert 951 543.817
insert 221 890.25
insert 895 -103.269
insert 404 623.077
insert 438 -257.489
insert -724 -290.655
maximum_key
print
empty
minimum_key
insert -172 -928.77
size
insert 360 -81.4216
size
insert 105 -644.852
maximum_key
size
insert -342 -926.787
print
insert -437 27.752
size
maximum_key
insert -455 -53.1746
quit
