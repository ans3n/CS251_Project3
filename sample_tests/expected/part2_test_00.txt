insert -909 378.732
insert -204 -9.91974
peek -909
378.732
insert 695 258.504
maximum_key
695
print
695 -> 258.504
└L: -204 -> -9.91974
 └L: -909 -> 378.732
size
3
print
695 -> 258.504
└L: -204 -> -9.91974
 └L: -909 -> 378.732
insert 297 268.386
empty
false
insert -481 -729.899
peek 297
268.386
insert 817 405.289
maximum_key
817
empty
false
insert 759 220.44
peek -481
-729.899
size
7
peek -909
378.732
size
7
minimum_key
-909
insert 849 -213.484
print
849 -> -213.484
└L: -481 -> -729.899
 ├L: -909 -> 378.732
 └R: 817 -> 405.289
  └L: 759 -> 220.44
   └L: 297 -> 268.386
    ├L: -204 -> -9.91974
    └R: 695 -> 258.504
insert -833 519.52
print
-833 -> 519.52
├L: -909 -> 378.732
└R: 849 -> -213.484
 └L: -481 -> -729.899
  └R: 817 -> 405.289
   └L: 759 -> 220.44
    └L: 297 -> 268.386
     ├L: -204 -> -9.91974
     └R: 695 -> 258.504
insert 876 875.023
maximum_key
876
print
876 -> 875.023
└L: 849 -> -213.484
 └L: -833 -> 519.52
  ├L: -909 -> 378.732
  └R: -481 -> -729.899
   └R: 817 -> 405.289
    └L: 759 -> 220.44
     └L: 297 -> 268.386
      ├L: -204 -> -9.91974
      └R: 695 -> 258.504
peek 297
268.386
insert -651 -184.926
insert -13 266.374
insert -746 -187.403
maximum_key
876
insert 513 -281.676
peek -833
519.52
size
14
maximum_key
876
print
876 -> 875.023
└L: 513 -> -281.676
 ├L: -833 -> 519.52
 │├L: -909 -> 378.732
 │└R: -746 -> -187.403
 │ └R: -13 -> 266.374
 │  ├L: -651 -> -184.926
 │  │└R: -204 -> -9.91974
 │  │ └L: -481 -> -729.899
 │  └R: 297 -> 268.386
 └R: 849 -> -213.484
  └L: 695 -> 258.504
   └R: 759 -> 220.44
    └R: 817 -> 405.289
insert -279 -10.7584
print
-279 -> -10.7584
├L: -746 -> -187.403
│├L: -833 -> 519.52
││└L: -909 -> 378.732
│└R: -651 -> -184.926
│ └R: -481 -> -729.899
└R: 513 -> -281.676
 ├L: -13 -> 266.374
 │├L: -204 -> -9.91974
 │└R: 297 -> 268.386
 └R: 876 -> 875.023
  └L: 849 -> -213.484
   └L: 695 -> 258.504
    └R: 759 -> 220.44
     └R: 817 -> 405.289
insert 951 543.817
insert 221 890.25
insert 895 -103.269
insert 404 623.077
insert 438 -257.489
insert -724 -290.655
maximum_key
951
print
951 -> 543.817
└L: 404 -> 623.077
 ├L: -724 -> -290.655
 │├L: -746 -> -187.403
 ││└L: -833 -> 519.52
 ││ └L: -909 -> 378.732
 │└R: -279 -> -10.7584
 │ ├L: -651 -> -184.926
 │ │└R: -481 -> -729.899
 │ └R: 221 -> 890.25
 │  ├L: -13 -> 266.374
 │  │└L: -204 -> -9.91974
 │  └R: 297 -> 268.386
 └R: 895 -> -103.269
  └L: 438 -> -257.489
   └R: 513 -> -281.676
    └R: 876 -> 875.023
     └L: 849 -> -213.484
      └L: 695 -> 258.504
       └R: 759 -> 220.44
        └R: 817 -> 405.289
empty
false
minimum_key
-909
insert -172 -928.77
size
22
insert 360 -81.4216
size
23
insert 105 -644.852
maximum_key
951
size
24
insert -342 -926.787
print
-342 -> -926.787
├L: -909 -> 378.732
│└R: -724 -> -290.655
│ ├L: -833 -> 519.52
│ │└R: -746 -> -187.403
│ └R: -481 -> -729.899
│  └L: -651 -> -184.926
└R: 105 -> -644.852
 ├L: -172 -> -928.77
 │├L: -279 -> -10.7584
 ││└R: -204 -> -9.91974
 │└R: -13 -> 266.374
 └R: 951 -> 543.817
  └L: 404 -> 623.077
   ├L: 360 -> -81.4216
   │└L: 297 -> 268.386
   │ └L: 221 -> 890.25
   └R: 895 -> -103.269
    └L: 438 -> -257.489
     └R: 513 -> -281.676
      └R: 876 -> 875.023
       └L: 849 -> -213.484
        └L: 695 -> 258.504
         └R: 759 -> 220.44
          └R: 817 -> 405.289
insert -437 27.752
size
26
maximum_key
951
insert -455 -53.1746
quit
